README.docker.md

```diff
@@ -0,0 +1,32 @@
+# Staytus
+
+Staytus is a complete solution for publishing the latest information about
+any issues with your web applications, networks or services. Along with
+absolutely beautiful public & admin interfaces, Staytus is a powerful tool for
+any organization with customers that rely on them to be online 24/7.
+
+* [Check out the live demo](http://demo.staytus.co)
+* [Read the roadmap](https://github.com/adamcooke/staytus/blob/master/ROADMAP.md)
+* [Report a bug](https://github.com/adamcooke/staytus/issues/new?labels=bug)
+* [Ask a question](https://github.com/adamcooke/staytus/issues/new?labels=question)
+
+![Screenshot](https://s.adamcooke.io/15/iOzvtk.png)
+
+## Setup with Docker
+
+1. [Install Docker locally](https://docs.docker.com/installation/)
+2. [Install docker-compose locally](https://docs.docker.com/installation/)
```

Comment:
These URLs should be different. You probably want https://docs.docker.com/compose/install/